autospec/specfiles.py

```diff
@@ -730,6 +730,46 @@ def write_maven_install(self):
         # All done
         self._write_strip("done")
 
+    def write_gradle_install(self):
+        """Write installation steps to specfile for maven source packages."""
+        self.write_license_files()
+
+        if self.subdir:
+            self._write_strip("pushd " + self.subdir)
+
+        self._write_strip("gradle --offline install -d")
+
+        # Enter the installation subdir
+        self._write_strip("pushd build/install")
+
+        # Handle jarfiles in lib dir
```

Comment:
I think this code comment could be expanded to describe in greater detail the rationale behind the code below that conditionally creates the jar symlinks.

Reply:
Adding comments